contracts/pcv/morpho/MorphoCompoundPCVDeposit.sol

```diff
@@ -238,6 +225,21 @@ contract MorphoCompoundPCVDeposit is PCVDeposit, ReentrancyGuard {
         }
     }
 
+    /// @notice set the pcv oracle address
+    /// @param _pcvOracle new pcv oracle to reference
+    function setPCVOracle(address _pcvOracle) external override onlyGovernor {
+        address oldOracle = pcvOracle;
+        pcvOracle = _pcvOracle;
+
+        _recordPNL();
+
+        IPCVOracle(pcvOracle).updateLiquidBalance(
+            lastRecordedBalance.toInt256()
+        );
+
+        emit PCVOracleUpdated(oldOracle, _pcvOracle);
+    }
+
```

Comment:
You must keep the default implementation here & don't create an override

`IPCVOracle(pcvOracle).updateLiquidBalance` will revert until we have no set an oracle for the PSM in the PCVOracle. So, we'll have to add the PSM "venue" in the PCV oracle first, and when we do this, the current balance of the PSM is read and we already store the PSM.balance() in the PCV Oracle's last balance